src/write.c

```diff
@@ -428,6 +429,7 @@ avifEncoder * avifEncoderCreate(void)
         return NULL;
     }
     memset(encoder, 0, sizeof(avifEncoder));
+    encoder->headerFormat = AVIF_HEADER_FULL;
```

Comment:
Nit: Move this line to the end of the function to match the declaration order of the struct members.